packages/browser/src/userfeedback.ts

```diff
@@ -0,0 +1,70 @@
+import { getCurrentHub } from '@sentry/core';
+import type { DsnComponents, EventEnvelope, SdkMetadata, UserFeedback, UserFeedbackItem } from '@sentry/types';
+import { createEnvelope, dsnToString, logger } from '@sentry/utils';
+
+/**
+ * Sends user feedback to Sentry.
+ */
+export function captureUserFeedback(feedback: UserFeedback): void {
+  const hub = getCurrentHub();
+  const client = hub.getClient();
+  const transport = client && client.getTransport();
+
+  if (!client) {
+    __DEBUG_BUILD__ && logger.log('[UserFeedback] getClient did not return a Client, user feedback will not be sent.');
+    return;
+  }
+
+  if (!transport) {
+    __DEBUG_BUILD__ &&
+      logger.log('[UserFeedback] getTransport did not return a Transport, user feedback will not be sent.');
+    return;
+  }
+
+  const envelope = createUserFeedbackEnvelope(feedback, {
+    metadata: client.getSdkMetadata && client.getSdkMetadata(),
+    dsn: client.getDsn(),
+    tunnel: client.getOptions().tunnel,
+  });
+
+  void transport.send(envelope);
```

Comment:
it's times like this I wish `client.sendEnvelope` was exposed - @lforst do you think we should just expose it?

We can only call `transport.send if `client.getOptions().enabled !== false`

Reply:
> m: We can only call transport.send if client.getOptions().enabled !== false

I have a feeling this should be checked in a more central location. Doesn't have to be this PR.

Reply:
> it's times like this I wish client.sendEnvelope was exposed - @lforst do you think we should just expose it?

Probably? It seems safer that this whole chain of grabbing stuff off the hub that may or may not be there. But honestly `sendEnvelope` seems a bit too low-level to just expose it on a pretty public place.

I vote we take the bundle size hit and go ahead and implement user feedback like we implemented sessions, errors, and transactions just to stay consistent - meaning we put it as a `captureUserFeedback()` method on the (browser-)client.

Reply:
Sounds good to me @krystofwoldrich - sorry for making you revert your work, but let's just add the methods to the browser client.

Reply:
I don't mind either, but the stand-alone function is a bit easier for reuse in other JS SDKs,

if we implement it on the Browser client we have to export the user feedback envelope creator for other JS-based SDKs, 

if we would implement it as a standalone function other JS SDKs can use the method right away.

Reply:
Both electron and react native wrap browser - so it's only the node SDKs that won't have access to this, and I think that is fine.

Reply:
Okay, that's fair not that many SDKs, just a note RN [doesn't wrap Browser](https://github.com/getsentry/sentry-react-native/blob/d7401ac44acf524fe4d9a6a525f2cc13c70ce9e5/src/js/client.ts#L34).